website/content/docs/api-gateway.mdx

```diff
@@ -0,0 +1,176 @@
+---
+layout: docs
+page_title: API Gateway
+description: Using Consul API gateway functionality
+---
+
+# Consul API Gateway
+
+This topic describes how to use the Consul API Gateway module, which enables applications and services running in a datacenter to be accessed by network clients outside the datacenter. This type of network traffic is commonly referred to as "north-south" network traffic.
+
+## Introduction
+
+Consul API Gateway is an implementation of the Kubernetes Gateway [API Specification](https://gateway-api.sigs.k8s.io/). The specification defines a set of custom resource definitions (CRD) that can create logical gateways and routes.
+
+## Requirements
+
+Your datacenter must meet the following requirements prior to configuring the Consul API Gateway:
+
+- A running Kubernetes cluster
+- Consul 1.11.2+
+
+## Installation
+
+1. Create a values file for your Consul helm chart definition that contains the following parameters:
+
+<CodeBlockConfig hideClipboard filename="values.yaml">
+
+   ```yaml
+   global:
+     name: consul
+     image: 'hashicorp/consul:1.11.2'
+     tls:
+       enabled: true
+   connectInject:
+     enabled: true
+   controller:
+     enabled: true
+   ```
+
+1. Install Consul API Gateway using the standard Consul Helm chart and specify the custom values file.
+
+```shell-session
+$ helm install consul hashicorp/consul --version 0.39.0 -f values.yaml
```

Comment:
```suggestion
$ helm install consul hashicorp/consul --version 0.39.0 --values values.yaml
```
A recent PR updated all of the example commands to use the long form for CLI flags per the recommendation from our style guide. I recommend changing this for consistency.